common/persistence/sql/sqlplugin/postgres/postgres_server_test.go

```diff
@@ -30,12 +32,17 @@ import (
 )
 
 const (
-	testUser      = "postgres"
-	testPassword  = "cadence"
 	testSchemaDir = "schema/postgres"
 )
 
 func getTestClusterOption() *pt.TestBaseOptions {
+	testUser := "postgres"
+	testPassword := "cadence"
+
+	if runtime.GOOS == "darwin" {
+		testUser = os.Getenv("USER")
```

Comment:
Could it use POSTGRES_USER?

Reply:
On OSX the default for postgresql user is USER, POSTGRES_USER is unset. Maybe you want me to use POSTGRES_USER with fallback to USER?